projects/dsp-ui/src/lib/viewer/operations/display-edit/display-edit.component.ts

```diff
@@ -114,6 +121,48 @@ export class DisplayEditComponent implements OnInit {
         }
     }
 
+    openDialog() {
+        const dialogData = new ConfirmationDialogData();
+        dialogData.title = 'Are you sure want to delete this value from ' + this.displayValue.propertyLabel + '?';
+        dialogData.message = 'Confirming this action will delete the following value from ' +
+                                this.displayValue.propertyLabel + ':<br/><br/>' + this._generateValueInfo();
+        dialogData.buttonTextOk = 'Yes, delete the value';
+        dialogData.buttonTextCancel = 'No, keep the value';
+
+        const dialogRef =
+            this._dialog.open<ConfirmationDialogComponent, ConfirmationDialogData>(ConfirmationDialogComponent, { data: dialogData});
+
+        dialogRef.afterClosed().subscribe((confirmed: boolean) => {
+            if (confirmed) {
+                this.deleteValue();
+            }
+        });
+    }
+
+    deleteValue() {
+        const deleteVal = new DeleteValue();
+        deleteVal.id = this.displayValue.id;
+        deleteVal.type = this.displayValue.type;
+
+        const updateRes = new UpdateResource();
+        updateRes.type = this.parentResource.type;
+        updateRes.id = this.parentResource.id;
+        updateRes.property = this.displayValue.property;
+        updateRes.value = deleteVal;
+
+        this._dspApiConnection.v2.values.deleteValue(updateRes as UpdateResource<DeleteValue>).pipe(
+        mergeMap((res: DeleteValueResponse) => {
+            // emit a ValueDeleted event to the listeners in resource-view component to trigger an update of the UI
+            this._valueOperationEventService.emit(new EmitEvent(Events.ValueDeleted, deleteVal));
+            return res.result;
+        })
+        ).subscribe(
+        () => {
+            // TODO: figure out what needs to be done here
```

Comment:
more details could be helpful in this comment otherwise the subscribe() should not be here

Reply:
This was leftover code, however I do still need to subscribe so that the emit method of `ValueOperationEventService` is called. I've removed the unnecessary function inside the subscribe in 18faf9f